stable/chat/v1/messages.proto

```diff
@@ -440,6 +414,8 @@ message Message {
   Content content = 7;
   // The reactions of the message.
   repeated Reaction reactions = 8;
+  // Bot actions to show on this message. Ignored if this is a button action.
```

Comment:
Why is it ignored if it's a button action?

Reply:
uhh, probably wrong copy-paste, will fix